### PR TITLE
Moving default attention in Phi to torch.nn.functional.scaled_dot_product_attention

### PR DESCRIPTION
**Pull request by blefaudeux**
- 20% faster on my laptop (3080)
- should take a lot less ram while training
- no big compromise, except for having a semi-recent torch installation

### CODE REVIEW

**Comment by blefaudeux** [comment]
Nothing changed in terms of accuracy, of course. No need to upcast and the like, sdpa will handle that
Screenshot from 2024-04-03 23-16-28
Screenshot from 2024-04-03 23-16-06

**Comment by vikhyat** [comment]
Thanks!
